include/libdnf/rpm/rpm_signature.hpp

```diff
@@ -99,6 +99,9 @@ class RpmSignature {
     /// @param key_url: URL of the public key to be imported.
     std::vector<KeyInfo> parse_key_file(const std::string & key_url);
 
+    /// Return string representation of the CheckResult enum
+    static std::string check_result_to_string(const CheckResult result);
```

Comment:
Parameter "result" is passed by value. const-qualification of parameters only has an effect in function definitions. The header file is for the user. It is unnecessary to tell him that the copy inside the implementation is a constant.